generate/parse.go

```diff
@@ -16,20 +16,30 @@ import (
 	"github.com/vektah/gqlparser/v2/validator"
 )
 
-func getSchema(filePatterns StringList) (*ast.Schema, error) {
-	sources, err := loadSchemaSources(filePatterns)
+func getSchema(globs StringList) (*ast.Schema, error) {
+	filenames, err := expandFilenames(globs)
 	if err != nil {
 		return nil, err
 	}
+
+	sources := make([]*ast.Source, len(filenames))
+	for i, filename := range filenames {
+		text, err := ioutil.ReadFile(filename)
+		if err != nil {
+			return nil, errorf(nil, "unreadable schema file %v: %v", filename, err)
+		}
+		sources[i] = &ast.Source{Name: filename, Input: string(text)}
+	}
+
 	schema, graphqlError := gqlparser.LoadSchema(sources...)
 	if graphqlError != nil {
-		filename, _ := graphqlError.Extensions["file"].(string)
-		return nil, errorf(nil, "invalid schema file %v: %v", filename, graphqlError)
+		return nil, errorf(nil, "invalid schema: %v", graphqlError)
```

Comment:
Removing file name from error message will make it really hard to find issues in schema or operation files. Since we still do not have strong IDE support for graphql SDL files.

WDYT?